static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
+// eslint-disable-next-line guardian-frontend/global-config
+const config = window.guardian.config;
+
+export default Object.assign(
+    {
+        hasTone(name) {
+            return (this.page.tones || '').indexOf(name) > -1;
```

Comment:
general code style question: for simple expressions like this one, could we opt for the arrow syntax? i.e.
```
hasTone: name => (this.page.tones || '').indexOf(name) > -1
```
feels a bit more readable.